### DISCUSSION TITLE
Can GDK UI be automated by pywinauto?

### DISCUSSION THREAD

**Discussion by simohauml** [pywinauto/pywinauto#1295] [General]
I am trying to automate an application created with GDK.
Amide is an opensource medical image viewer. It seems only top window can be detected, any other controls, such as buttons, icons, or even new windows or dialogs cannot be detected.
Is it possible for pywinauto to automate GDK application?

**Comment by vasily-v-ryabov** [comment]
Unfortunately GDK on Windows doesn't support any accessibility technology at all. Linux GDK app can be automated in branch `atspi` (not released yet).
